### PR TITLE
Shiplist

### PR DESCRIPTION
**Pull request by Tjoosten**
# PR #149

15b2325 - Sync my branch.
15c61d1 - Set namespaces to use statements.
9d7d34d - Clean up use of the missing namespace.
f5557e4 - Remove `.png` files - See #133

### CODE REVIEW

**Comment by Tjoosten** [comment]
@pietercolpaert due a lock of time i can't extend this PR anymore. Can you merge it plz

File: resources/lang/de/client.php [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "rideEarlier" => "frühere",
     "rideLater" => "spätere",
     "latestRide" => "Letzte",
+    "latestRide" => "Letzte",
```
Comment (pietercolpaert) [inline resources/lang/de/client.php]: ?

File: app/Http/Controllers/ClassicRedirectController.php [hunk 2]
```diff
@@ -5,6 +5,11 @@
 use Illuminate\Support\Facades\Redirect;
 use Illuminate\Support\Facades\Config;
 use Illuminate\Http\Request;
+<<<<<<< HEAD
+use App\Http\Requests;
+use hyperRail\StationString;
+=======
+>>>>>>> iRail/development
```
Comment (pietercolpaert) [inline app/Http/Controllers/ClassicRedirectController.php]: This is a merge conflict which needs to be resolved

File: app/Http/Controllers/StationController.php [hunk 3]
```diff
@@ -295,7 +296,12 @@ public function specificTrain($station_id, $liveboard_id)
      */
     public function liveboard($id)
     {
+<<<<<<< HEAD
```
Comment (pietercolpaert) [inline app/Http/Controllers/StationController.php]: merge conflict

Comment (Tjoosten) [inline app/Http/Controllers/StationController.php]: fixed now normally.

Comment (pietercolpaert) [inline app/Http/Controllers/StationController.php]: no

File: app/Http/Controllers/StationController.php [hunk 4]
```diff
@@ -295,7 +296,12 @@ public function specificTrain($station_id, $liveboard_id)
...
+
+        $guzzleClient = new Client();
+=======
         $guzzleClient = new \GuzzleHttp\Client();
+>>>>>>> iRail/development
```
Comment (pietercolpaert) [inline app/Http/Controllers/StationController.php]: merge conflict

File: app/Http/Controllers/StationController.php (hunk 3 above)
Comment (Tjoosten) [inline app/Http/Controllers/StationController.php]: Crap my fault;

File: app/Http/Middleware/Language.php [hunk 5]
```diff
@@ -17,6 +17,7 @@ class Language
         'nl',
         'en',
         'fr',
+        'de',
```
Comment (pietercolpaert) [inline app/Http/Middleware/Language.php]: Didn't we agree on not enabling this right now?

**Comment by Tjoosten** [comment]
All fixed

File: resources/views/language.blade.php [hunk 6]
```diff
@@ -18,6 +18,7 @@
                 <li><a href="route?lang=nl">Nederlands</a></li>
                 <li><a href="route?lang=fr">Français</a></li>
                 <li><a href="route?lang=en">English</a></li>
+                <li><a href="route?lang=de">Deutsch</a></li>
```
Comment (pietercolpaert) [inline resources/views/language.blade.php]: This is still too much?

Comment (Tjoosten) [inline resources/views/language.blade.php]: Urgh plz kick me.

**Comment by pietercolpaert** [comment]
@Tjoosten will you pull request the deleted images to https://github.com/iRail/iRail-images?

**Comment by Tjoosten** [comment]
Yes. ASAP?

**Comment by pietercolpaert** [comment]
Whenever you find the time :)

**Comment by Tjoosten** [comment]
I will bring them up this evening

**Pull request merged**